source/StartConditionsPanel.cpp

```diff
@@ -148,7 +156,8 @@ bool StartConditionsPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &c
 
 		Select(startIt);
 	}
-	else if(startIt != scenarios.end() && (key == 's' || key == 'n' || key == SDLK_KP_ENTER || key == SDLK_RETURN))
+	else if(startIt != scenarios.end() && (key == 's' || key == 'n' || key == SDLK_KP_ENTER || key == SDLK_RETURN)
+		&& startIt->Unlocked(GameData::GlobalConditions()))
```

Comment:
This means that if a start has `to reveal` false but `to unlock` true, the start can still... start. Are you able to instead check if the info condition "unlocked start" is true?

Reply:
```suggestion
		&& info.HasCondition("unlocked start"))
```

Reply:
like this?